app/scripts/views/ready.js

```diff
@@ -82,10 +87,15 @@ define(function (require, exports, module) {
         headerId: this._getHeaderId(),
         isSync: this.relier.isSync(),
         redirectUri: this.relier.get('redirectUri'),
-        secondaryEmailVerified: this.getSearchParam('secondary_email_verified') || null
+        secondaryEmailVerified: this.getSearchParam('secondary_email_verified') || null,
+        serviceRedirect: this.model.get('serviceRedirect')
       });
     },
 
+    redirectToRelier () {
+      this._window.location.href = this.model.get('serviceRedirect');
```

Comment:
To ensure the metrics are flushed, you should call `this.navigateAway(this.model.get('serviceRedirect');`. Hopefully Chrome won't block if the location.href happens in a promise, if it does then maybe trigger a `this.metrics.flush()` then update the href.